bin/dp-cli.js

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+const { Command } = require('commander')
+const chalk = require('chalk')
+const createSite = require('../src/commands/createSite')
+const registerActor = require('../src/commands/registerActor')
+const sendPost = require('../src/commands/sendPost')
+const generateKeypairCommand = require('../src/commands/generateKeypair')
+const setAuthTokenCommand = require('../src/commands/setAuthToken')
+
+const program = new Command()
+
+program
+  .name('dp-cli')
+  .description('CLI for interacting with Distributed Press API and Social Inbox')
+  .version('1.0.0')
+
+// Define commands
+program
+  .command('create-site')
+  .description('Create a new site on Distributed Press')
+  .action(() => {
+    createSite()
```

Comment:
Have optional CLI args for the site info so we can skip asking from the TUI in case its part of a script